doc/nrf/libraries/networking/lwm2m_location_assistance.rst

```diff
@@ -41,6 +41,85 @@ There are three different supported methods of obtaining the location assistance
+
+Cell-based location
+===================
+
+Cell-based location uses only on current cell or current and neighboring cells.
+To get information about the current cell, use the Connectivity Monitor object (ID 4).
+When you enable the Kconfig option :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_CONN_MON_OBJ_SUPPORT`, the connectivity monitor is populated with data about the current cell.
+
+To get information about the neighboring cells, use a collection of ECID-Signal Measurement Information objects (ID 10256).
+To populate the objects, call the :c:func:`lwm2m_ncell_handler_register` function to register the listener for the neighborhood measurements and :c:func:`lwm2m_ncell_schedule_measurement` to schedule a measurement.
+
+The Ground Fix Location object needs to address the ``report_back`` resource before sending a location request.
+Back reporting tells the server whether it needs to send the acquired location back to the device.
+If the location is sent back to the device, the location is stored only in the Ground Fix Location object.
+
+To send the location request for the cell-based location, call the :c:func:`location_assistance_ground_fix_request_send` function.
+
+A-GPS location
+==============
+
+When using A-GPS location, the device requests A-GPS assistance data from the server.
+You can query the GNSS module for the data needed.
+A device can request for all data at once or split the request to reduce the memory usage.
+The request also contains information about the current cell the device is connected to and the information is similarly available on Connectivity Monitor object as in the cell-based location.
+
+When requesting for A-GPS assistance data, the device must first set the mask for the data it is requesting by calling the :c:func:`location_assistance_agps_set_mask` function.
+When the mask has been set, the :c:func:`location_assistance_agps_request_send` function sends the request with all necessary data to the server and responds with the A-GPS assistance data.
+The assistance data is written to the GNSS module automatically by the library.
+
+Filtered A-GPS
+--------------
+
+There is also an option to use filtered A-GPS.
+With filtered A-GPS, the satellites below the given angle above the ground are be filtered out.
+You can set the the angle to a degree `[0 - 90]` using the :c:func:`location_assist_agps_set_elevation_mask` function.
+Setting the degree to `-1` disables filtering, which is the default setting.
+
+P-GPS location
+==============
+
+When using P-GPS location, the device requests predictions for the satellites for a near future.
+P-GPS does not use information about current cell at all.
+It stores the information about satellites and injects the data to the GNSS module when needed.
+When using P-GPS, external flash is necessary as each prediction needs 2 kB of memory.
+
+When requesting for P-GPS assistance data, the device can set the P-GPS resources.
```

Comment:
Does a P-GPS request with default values work properly? The request specifies the date and time for the first prediction, so I don't know what happens if you don't set that.

Reply:
Yes it does. If you leave the starting day and starting time as 0, nRFCloud will treat them as current day and time.